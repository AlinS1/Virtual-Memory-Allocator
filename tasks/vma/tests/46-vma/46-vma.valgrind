==1766== Memcheck, a memory error detector
==1766== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==1766== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==1766== Command: /mnt/c/Users/alins/OneDrive/Desktop/SDA_tema1/vma
==1766== Parent PID: 1765
==1766== 
==1766== 
==1766== HEAP SUMMARY:
==1766==     in use at exit: 0 bytes in 0 blocks
==1766==   total heap usage: 13 allocs, 13 frees, 1,248 bytes allocated
==1766== 
==1766== All heap blocks were freed -- no leaks are possible
==1766== 
==1766== For lists of detected and suppressed errors, rerun with: -s
==1766== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
