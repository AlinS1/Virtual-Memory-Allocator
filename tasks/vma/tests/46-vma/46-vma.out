Perm:2
gigel
